README.md

```diff
@@ -191,13 +107,31 @@ See test README for test file structure.
   Run methods being tested through system tests only because they depend on, and affect the overall system.
   Constructors cannot be tested without getters which most classes don't implement.
 
+### Iteration 2
+- The scheduler algorithm prioritizes the prevention of starvation over achieving
+the highest throughput possible.
+
+  - Throughout: A measure of efficiency. The rate at which the elevator control system can process
+passenger destination requests within a specified amount of time. 
+
+  - Starvation: A passenger can experience starvation if they are left waiting for an elevator
+for an extended period of time. This would happen if they are on a floor that is rarely used and far away.
```

Comment:
"for an extended period of time." -> avoid this for now, we don't actually have a defined deadline.
Starvation prevention is about not having a case where it's possible the elevator never gets to a passenger.